config/tsconfig.json

```diff
@@ -14,6 +14,7 @@
     "strict": true,
     "target": "es2020",
     "lib": ["ES2020", "DOM"],
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "useUnknownInCatchVariables": false
```

Comment:
Ah, this needs to be the otheway around: we *want* to have `unknown` there, atm this now goes to `any`:

<img width="491" alt="grafik" src="https://github.com/user-attachments/assets/10966185-40a5-42fc-845c-c141516b6f64">

So and this is the version of when removing the new assignment with `e` still being `any`:

<img width="464" alt="grafik" src="https://github.com/user-attachments/assets/9cbd506f-7e00-4f6e-933a-9b5cfe4da353">

So no underlying of `e.message` (what we would want. `any` is just saying: use whatever you want, I do not care! 🙂  

With `unknown` is the more strict version - indicating that for `e` we just don't (and can't) know the type.

Ah, just seeing that a plain use in `new Error()` is then not directly possible. 🤔 

<img width="427" alt="grafik" src="https://github.com/user-attachments/assets/a19e14ae-3111-4769-a04c-8fde4324c318">
<img width="779" alt="grafik" src="https://github.com/user-attachments/assets/80e7eb57-1f46-4383-a0aa-ae909301662b">

But for `any` we still have typed as `any`. Also not optimal:

<img width="477" alt="grafik" src="https://github.com/user-attachments/assets/341d9efa-f462-4758-b370-a36bd9e998ef">

Hmm. Let me give this some thought, input welcome from the team.